data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report6_fitdata.py

```diff
@@ -17,30 +17,40 @@
 
 import urllib
 import pandas as pd
-
-from common import PIPELINE_TABLES
+from common import JINJA_ENV
 from utils import auth
 from gcloud.bq import BigQueryClient
 from analytics.cdr_ops.notebook_utils import execute, IMPERSONATION_SCOPES
 pd.options.display.max_rows = 120
 
 # + tags=["parameters"]
 project_id = ""
-pipeline = ""
-non_deid_fitbit = ""
-deid_cdr_fitbit = ""
-deid_cdr = ""
-com_cdr = ""
-truncation_date = ""
-maximum_age = ""
-run_as = ""
+pipeline=""
+non_deid_fitbit=""
+deid_cdr_fitbit=""
+deid_cdr=""
+com_cdr=""
+truncation_date=""
+maximum_age=""
 fitbit_sandbox_dataset = ""
 sleep_level_sandbox_table = ""
 fitbit_dataset = ""
-# -
+run_as=""
 
-# df will have a summary in the end
-df = pd.DataFrame(columns=['query', 'result'])
+# +
+# Parameters
+#project_id = "aou-res-curation-prod"
```

Comment:
This parameter should be removed, we do not want our dataset and project names in the git history.